### PR TITLE
[Misc][Tapayan] Finished Header Component

### PR DESCRIPTION
**Pull request by tapsTapayan**

### CODE REVIEW

File: styles/_theme.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+@use "./constants" as constants;
+
+@mixin SET_BACKGROUND($color, $alpha) {
+    background-color: rgba($color, $alpha);
+}
```
Comment (jdesolate) [inline styles/_theme.scss:5]: Pls add new line at end of the file

File: components/Styled_Button/index.tsx [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+} 
+
+const Styled_Button = (params: Button_Params) => {
+  const {children, types, themes} = params;
+  return (
+    <Button className={style[types!]}><h1>{children}</h1></Button>
+  )
+}
+
+export default Styled_Button
```
Comment (jdesolate) [inline components/Styled_Button/index.tsx:18]: Please add new line at end of the file

File: components/Styled_Button/_index.module.scss [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+    color: constant.$COLOR_SECONDARY_DARK;
+    padding: 5px 10px;
+
+    h1{
+        @include text.BUTTON_TEXT_SMALL;
+        font-family: constant.$FONT_MAIN;
+        font-style: normal;
+        font-weight: 600;
+    }
+}
```
Comment (jdesolate) [inline components/Styled_Button/_index.module.scss:31]: Please add new line at end of the file

**Review by jdesolate — APPROVED** [review]
Pre-approved with minor comments

File: components/Styled_Button/index.tsx (hunk 2 above)
Comment (tapsTapayan) [reply components/Styled_Button/index.tsx:18]: Noted - Taps

File: components/Styled_Button/_index.module.scss (hunk 3 above)
Comment (tapsTapayan) [reply components/Styled_Button/_index.module.scss:31]: Noted - Taps

**Pull request merged**